tgui-next/packages/tgui/interfaces/NtosSupermatterMonitor.js

```diff
@@ -61,10 +61,10 @@ export const NtosSupermatterMonitor = props => {
                 minValue={0}
                 maxValue={logScale(10000)}
                 ranges={{
-                  teal: [-Infinity, logScale(80)],
-                  good: [logScale(80), logScale(373)],
-                  average: [logScale(373), logScale(1000)],
-                  bad: [logScale(1000), Infinity],
+                  teal: [-Infinity, logScale(80)] / logScale(10000),
+                  good: [logScale(80) / logScale(10000), logScale(373) / logScale(10000)],
```

Comment:
This is also a major downside, because you have to do manual scaling on the range side.